src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -278,9 +278,7 @@ class PillSelectorBase extends React.Component<Props, State> {
                         {...rest}
                         {...inputProps}
                         autoComplete="off"
-                        className={classNames('bdl-PillSelector-input', 'pill-selector-input', className, {
-                            'bdl-PillSelector-input--showAvatars': showAvatars,
```

Comment:
not sure what this class was intended for but there's not reference of it in BUIE or GHE